Index: build.properties.default
===================================================================
--- build.properties.default	(revision 1826375)
+++ build.properties.default	(revision 1826376)
@@ -96,12 +96,12 @@
 base-tomcat.loc.1=${base-apache.loc.1}/tomcat
 base-tomcat.loc.2=${base-apache.loc.2}/tomcat
 
-base-sf.loc=http://downloads.sourceforge.net
+base-sf.loc=https://downloads.sourceforge.net
 # repo.maven.apache.org is the same as repo2.maven.org
-base-maven.loc=http://repo.maven.apache.org/maven2
+base-maven.loc=https://repo.maven.apache.org/maven2
 
 # Mirror, was used when there were problems with the main SF downloads site
-# base-sf.loc=http://sunet.dl.sourceforge.net
+# base-sf.loc=https://sunet.dl.sourceforge.net
 
 # ----- Webservices - JAX RPC -----
 jaxrpc-lib.version=1.1-rc4
